### ISSUE TITLE
If time permits: change color of selection menu / tabs

### ISSUE THREAD

**Issue by WHOequity** [WHOequity/who-heat#261]
At the moment, it's quite hard to see which menu item or tab is selected. In the main menu at the top, selected menu items appear in a dark blue (instead of a light blue) and in the sub menu, selected tabs appear with a border (instead of without a border). Is it possible to change this? 

Is it, for example, possible to have selected menu items appear in a different color (e.g. orange, similar to what we see on the who website - see screenshot below). And what options do we have for the tabs? 

![untitled](https://cloud.githubusercontent.com/assets/13563065/12778664/30d04512-ca64-11e5-888e-c8f66e75e4a1.png)

**Comment by zross** [comment]
I agree with this. Changing the highlight color can be done relatively easily. For the tabs, what about just having them be pull downs

![image](https://cloud.githubusercontent.com/assets/1395177/12783595/74cb0874-ca50-11e5-8f33-d017ef3145eb.png)

We could also have them be some kind of button like the right side of the page you sent.

**Comment by WHOequity** [comment]
The orange color for selected menu items looks great. If possible, the color should also change to orange when hovering over other menu items (same as on WHO website). 

The pull down menu for "About" also looks great. Yes, please implement for all menu items (with the pull down menu opening up on hover). 
 
Can you also please
- change the info sign back to "About" text,
- increase the font size for items in the pull down menu, 
- change the font color to white when hovering over pull down menu items (background then appears dark blue and the black text is hard to read).

**Comment by WHOequity** [comment]
An additional note for the "Explore Inequality" and "Compare Inequality tabs: 

If we have a pull down menu instead of tabs, users won't be able to see which "tab" they are in. Can we add the "tab" name to the title in the left panel. For example, if the user selects "Disaggregated data (tables)", the title in the left panel should read: 
Explore Inequality 
Disaggregated data (tables)

**Comment by WHOequity** [comment]
The main menu and tabs look good now. Two comments: 
- Can you please increase the font size for "License" in the drop down menu for "About" (currently it's a bit smaller than the other items)?
- Can you please add a tab under "About" for "Feedback"?
